### ISSUE TITLE
unable to upload from Fuji V100 camera (bug report attached)

### ISSUE THREAD

**Issue by ericaquaronne** [damonlynch/rapid-photo-downloader#239]
failed to upload from my Fuji x100V 
this is a regression as it used to work, I then updated my ubuntu to the latest LTS and bang, this brakes, (same pb with Digikam broken since update)
I updated to the latest ubuntu LTS updates before doing the test
[rpd-bug-report-20241206.tar.gz](https://github.com/user-attachments/files/18038969/rpd-bug-report-20241206.tar.gz)

**Comment by damonlynch** [comment]
This is a problem with libgphoto2, which both Rapid Photo Downloader and Digikam (and every other Linux application) use to access files on your camera.

1. Install the command line application gPhoto2, if it is not already installed
2. Use gPhoto2 to list the files on the memory card in the camera, and attempt to download them
3. This will almost certainly fail
4. File a [bug report with the libgphoto2 developers](https://github.com/gphoto/libgphoto2/issues)

Simply put: if you do not do this, the problem will not be fixed. If you have any questions, let me know. Otherwise, I will close this bug report in a few days.
